src/main/java/mate.academy/Main.java

```diff
@@ -1,17 +1,7 @@
 package mate.academy;
 
-import java.util.List;
-import mate.academy.lib.Injector;
-import mate.academy.model.Product;
-import mate.academy.service.ProductService;
-
 public class Main {
```

Comment:
This class shouldn't be empty, please add again what was here and test your injector here. Previous version of this file:

```java
public static void main(String[] args) {
        // Please test your Injector here. Feel free to push this class as a part of your solution
        Injector injector = Injector.getInjector();
        ProductService productService = null;
        List<Product> products = productService.getAllFromFile("products.txt");
        products.forEach(System.out::println);

    }
```